wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/impl/resolver/URIResolver.java

```diff
@@ -549,5 +551,13 @@ public NamespaceContext getNamespaceContext(Document doc) {
     private final String[] errors = new String[] {
                                     "Can not resolve reference type",
                                     "Required SOAPMessage instance to resolve reference"};
+
+    public XMLSignatureInput engineResolveURI(ResourceResolverContext resourceResolverContext) throws ResourceResolverException {
+        return null;
```

Comment:
it is always needed to nullify the result?

Reply:
```suggestion
        return engineResolve(rrc.attr, rrc.baseUri);
```